clippy.toml

```diff
@@ -1 +1,2 @@
-doc-valid-idents = ["PostgreSQL", "MySQL", "SQLite", "JavaScript", "OpenAPI"]
+avoid-breaking-exported-api = false
```

Comment:
Why was this lint disabled?

Reply:
Putting `false` here enables more lints, not disables them.

Reply:
Oh right, I misread the clippy docs, disregard then, everything's fine!